qaframework-bdd-tests/src/test/resources/features/refapp-2.x/07-visit/visitTypes.feature

```diff
@@ -1 +1,39 @@
-Feature: Visit types Management
\ No newline at end of file
+Feature: Visit types Management
+
+  Background:
+    Given a user clicks on the system administration link from the home page
+    Then the system loads the administration page
+    And a user clicks on the manage Visit Types link
+
+  @selenium
+  @visitType
+  Scenario: Adding a Visit Type
+    When a user clicks on the add visit type link on the manage visit type page
+    Then the system loads the visit type form page
+    And a user fills the add visit type form
+    And a user saves visit type
+    Then a visit type is saved successfully
+
+  @selenium
+  @visitType
+  Scenario: Editing a Visit Type Test
+    When a user clicks on the already saved visit type link
+    Then the system loads the visit type form page
+    And a user edits the visit type name
+    Then a visit type is saved successfully
+
+  @selenium
+  @visitType
+  Scenario: Retiring a Visit Type Test
+    When a user clicks on the already saved visit type link
+    Then the system loads the visit type form page
+    And a user fills in the reason for retiring visit type
+    Then a user clicks on retire visit type button
+
+  @selenium
+  @visitType
+  Scenario: Deleting a Visit Type Test
+    When a user clicks on the already saved visit type link
+    Then the system loads the visit type form page
```

Comment:
is visit type a form or a page?

Reply:
>https://qa-refapp.openmrs.org/openmrs/admin/visits/visitType.form
According to url it is a form ..but thought always ...we call them pages

Reply:
> > https://qa-refapp.openmrs.org/openmrs/admin/visits/visitType.form
> > According to url it is a form ..but thought always ...we call them pages


@Muta-Jonathan is it possible to be both a form and a page?